test/test-custom-galleon/my/custom/galleon/settings.xml

```diff
@@ -0,0 +1,5 @@
+<settings xmlns="http://maven.apache.org/settings/1.0.0"    
```

Comment:
Minor and probably would work either way, however, according to https://maven.apache.org/settings.html, the `SETTINGS` should be uppercase
```
    <settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
```

Reply:
@jliznak, Thank-you. Fixed.